### PR TITLE
Add camunda setup

### PR DESCRIPTION
**Pull request by withrocks**
Script and config for setting up Camunda locally.

### CODE REVIEW

File: middleware/camunda/local-server.xml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    <Resource name="jdbc/ProcessEngine"
+              auth="Container"
+              type="javax.sql.DataSource"
+              factory="org.apache.tomcat.jdbc.pool.DataSourceFactory"
+              uniqueResourceName="process-engine"
+              driverClassName="org.postgresql.Driver"
+              url="jdbc:postgresql:clims"
+              defaultTransactionIsolation="READ_COMMITTED"
+              username="camunda"
+              password="changeit"
```
Comment (johandahlberg) [inline middleware/camunda/local-server.xml]: Don't know, but perhaps we should generate a random password on each machine instead?

**Comment by johandahlberg** [comment]
Looks good. Just one minor comment, as you can se above.

File: middleware/camunda/local-server.xml (hunk 1 above)
Comment (withrocks) [reply middleware/camunda/local-server.xml]: This is just for development purposes, as the name of the file implies (local). I wonder if we should wait for that until we create the docker images?

Comment (withrocks) [reply middleware/camunda/local-server.xml]: I'll fix it :)

**Review by johandahlberg — APPROVED** [review]

**Pull request merged**